test/e2e/e2e_test.go

```diff
@@ -167,24 +170,144 @@ func TestLowNodeUtilization(t *testing.T) {
 	}
 
 	var nodes []*v1.Node
+	var workerNodes []*v1.Node
 	for i := range nodeList.Items {
 		node := nodeList.Items[i]
 		nodes = append(nodes, &node)
+		if _, exists := node.Labels["node-role.kubernetes.io/master"]; !exists {
+			workerNodes = append(workerNodes, &node)
+		}
 	}
 
+	t.Log("Creating testing namespace")
 	testNamespace := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "e2e-" + strings.ToLower(t.Name())}}
 	if _, err := clientSet.CoreV1().Namespaces().Create(ctx, testNamespace, metav1.CreateOptions{}); err != nil {
 		t.Fatalf("Unable to create ns %v", testNamespace.Name)
 	}
 	defer clientSet.CoreV1().Namespaces().Delete(ctx, testNamespace.Name, metav1.DeleteOptions{})
 
-	rc := RcByNameContainer("test-rc-node-utilization", testNamespace.Name, int32(15), map[string]string{"test": "node-utilization"}, nil, "")
+	// Make all worker nodes resource balanced
+	cleanUp, err := testutils.CreateBalancedPodForNodes(t, ctx, clientSet, testNamespace.Name, workerNodes, 0.5)
+	if err != nil {
+		t.Fatalf("Unable to create load balancing pods: %v", err)
+	}
+	defer cleanUp()
+
+	t.Log("Creating pods each consuming 10% of node's allocatable")
+	nodeCpu := workerNodes[0].Status.Allocatable[v1.ResourceCPU]
+	tenthOfCpu := int64(float64((&nodeCpu).MilliValue()) * 0.1)
+
+	t.Log("Creating pods all bound to a single node")
```

Comment:
@damemi @seanmalloy I did not like the way TestLowNodeUtilization was tested. The threshold of deciding when a node is overutilized and underutilized was quite thin. Now, the idea is to resource balance all worker nodes first (all eating 50% of resources) and then overutilizing the first worker node. Once LowNodeUtilization is ran, the test verifies at least one pod was evicted from the overutilized node. Open to more improvements.

Reply:
Making sure I understand this, you're doing:
1. Balance nodes at (at least) 50% usage using `createBalancedPodForNodes`
2. Create 5 more pods on 1 node, bringing that node to 100% usage
3. Run LNU with `lowThreshold: 70%` and `highThreshold: 80%`

My only feedback is maybe we should be careful about trying to fill the node up to 100%, if it's running on a high-usage cluster then this test might balance the pods at higher than 50%, then the test will fail when you can't create all 5 pods.

Reply:
That's a good point. Decreased to 4 pods.